modules/src/core/ics02_client/msgs/create_client.rs

```diff
@@ -16,17 +15,13 @@ pub const TYPE_URL: &str = "/ibc.core.client.v1.MsgCreateClient";
 /// A type of message that triggers the creation of a new on-chain (IBC) client.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct MsgCreateAnyClient {
-    pub client_state: AnyClientState,
+    pub client_state: Any,
```

Comment:
Rust is giving me the following error here:
>the trait bound `ibc_proto::google::protobuf::Any: core::cmp::Eq` is not satisfied
the trait `core::cmp::Eq` is not implemented for `ibc_proto::google::protobuf::Any`

I'm not sure why?